### ISSUE TITLE
Chart unexpectedly pans when clicked

### ISSUE THREAD

**Issue by JonathanWylie** [amcharts/amcharts5#455]
If you have an XY chart with panning enabled and a click handler registered, then a single click causes the graph to pan a small amount, even though no panning was actually done. If the click handler is not registered, then clicking on the chart does not cause the pan to happen.

Here is a demo showing the problem: https://codesandbox.io/s/jolly-butterfly-pgzoso

**Comment by zeroin** [comment]
We will investigate this, meanwhile I can recommend adding click listener to some a chartContainer instead:
```
    chart.chartContainer.events.on("click", (event) => {
      alert("chart clicked");
    });
```

**Comment by martynasma** [comment]
Fixed in [5.2.8](https://github.com/amcharts/amcharts5/releases/tag/5.2.8).

## [5.2.8] - 2022-06-15

### Added
- Private (read-only) setting `tooltipPosition` added to `Axis`.
- `"line-through"` label decoration added. Works both via `Label`'s `textDecoration` setting, and in-line, e.g. `This is [line-through]a crossed out text[/]`.
- `autoScale` (default: `false`) setting added to `MapPointSeries`. If set to `true`, bullets will resize together with a map while zooming.
- `PieSeries` and `FunnelSeries` now properly support bullets.

### Fixed
- Changing randius or innerRadius on a PieChart was not working.
- Changing legendValueText or legendLabel text on PercentSeries was not working.
- `exportfinished` event was not kicking in.
- Exporting plugin will now try to finish any ongoing rendering before genrating export image.
- Using mouse wheel on a non-chart element that is positioned over the chart works properly now.
- If some non-chart element was rendered over `Scrollbar`, doublicking on a thumb area would still zoom it out.
- Adding `click` event to an `XYChart` was causing it to pan a little when clicked.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
